cider/tests/utils/DuckDbQueryRunner.cpp

```diff
@@ -416,15 +411,14 @@ void DuckDbResultConvertor::updateChildrenNullCounts(CiderBatch& batch) {
 
 CiderBatch DuckDbResultConvertor::fetchOneArrowFormattedBatch(
     std::unique_ptr<duckdb::DataChunk>& chunk,
-    std::vector<std::string>& names,
-    std::string config_timezone) {
+    std::vector<std::string>& names) {
   int col_num = chunk->ColumnCount();
   int row_num = chunk->size();
 
   // Construct ArrowSchema using methods from duckdb
   std::vector<::duckdb::LogicalType> types = chunk->GetTypes();
   auto arrow_schema = CiderBatchUtils::allocateArrowSchema();
-  ::duckdb::QueryResult::ToArrowSchema(arrow_schema, types, names, config_timezone);
+  DuckDbArrowSchemaAdaptor::duckdbResultToArrowSchema(arrow_schema, types, names);
 
```

Comment:
can move `updateChildrenNullCounts` into this method and update inside?

Reply:
I'm afraid the answer is no. `null_count` is an attribute of arrow **array**, but we are dealing with arrow **schema** here, and arrow schema does not have a `null_count` attribute.